### PR TITLE
Changed get_cats.html to personal_cats.html

### PR DESCRIPTION
**Pull request by FBastiaan04**
The `index.html` file had a link to `get_cats.html` instead of `personal_cats.html`

fba257

### CODE REVIEW

File: http_server_check/data/index.html [hunk 1]
```diff
@@ -5,13 +5,13 @@
     <body>
         <h1>Computer Networks Server</h1>
```
Comment (fzovpec) [inline http_server_check/data/index.html:6]: Please do not change file lines not related to your PR

File: http_server_check/data/index.html [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
-            If you can see this line in your browser, you already worked with HTTP more than most of the programmers ever did or will do! 
-            We, Computer Networks Team, want to congratulate you with this archivement! 
+        <p>
+            If you can see this line in your browser, you already worked with HTTP more than most of the programmers ever did or will do!
+            We, Computer Networks Team, want to congratulate you with this archivement!
         </p>
 
         <p>
-            Apart from the assignement points, we want to award you with two cutest cat images we could find. 
+            Apart from the assignement points, we want to award you with two cutest cat images we could find.
```
Comment (fzovpec) [inline http_server_check/data/index.html:14]: This one too

**Review by fzovpec — CHANGES_REQUESTED** [review]
Hi there! Thank you very much for your contribution. Your PR is good and I have decided to award you with 50 points for your effort! However, before doing so, I would ask you to make some changes to your PR. Please delete an ./idea/workspace.xml file from the commit and do not change file lines not related to your PR. Please do so by Wednesday 10th 23:59

**Comment by FBastiaan04** [comment]
> Hi there! Thank you very much for your contribution. Your PR is good and I have decided to award you with 50 points for your effort! However, before doing so, I would ask you to make some changes to your PR. Please delete an ./idea/workspace.xml file from the commit and do not change file lines not related to your PR. Please do so by Wednesday 10th 23:59

I don't know how I missed the .idea folder. I created a [new pull request](https://github.com/atlarge-research/cn-lab-student/pull/19) which should only edit the line in question.

**Pull request closed without merging**